pkg/i18n/i18n.go

```diff
@@ -38,6 +38,10 @@ func NewTranslationSet(log *logrus.Entry) *TranslationSet {
 		mergo.Merge(&set, germanSet(), mergo.WithOverride)
 	}
 
+	if strings.HasPrefix(userLang, "tr") {
+		mergo.Merge(&set, turkishSet(), mergo.WithOverride)
```

Comment:
@jesseduffield  Whats wrong? how can i fix this? I just did copy / paste :)

Reply:
Can you change all of these to: `_ = mergo.Merge(....)`,  
That way the golangcibot will stop telling this.  
We hadn't setup the golangcibot before your pr so that's why this hasn't been fixed yet.